config/deploy.rb

```diff
@@ -0,0 +1,43 @@
+# config valid for current version and patch releases of Capistrano
+lock "~> 3.17.1"
+
+set :application, "dlme"
+set :repo_url, "https://github.com/sul-dlss/dlme.git"
+
+# Default branch is :master
+ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
+
+# Default deploy_to directory is /var/www/my_app_name
+set :deploy_to, "/opt/app/dlme/dlme"
+
+# Default value for :format is :airbrussh.
+# set :format, :airbrussh
+
+# You can configure the Airbrussh format using :format_options.
+# These are the defaults.
+# set :format_options, command_output: true, log_file: "log/capistrano.log", color: :auto, truncate: :auto
+
+# Default value for :pty is false
+# set :pty, true
+
+# Default value for :linked_files is []
+set :linked_files, %w[config/database.yml]
```

Comment:
we also linked `config/blacklight.yml` for VT, but that might not matter?

Reply:
That doesn't make sense unless you're also using shared_configs.

Reply:
Ah, I wondered about that. thx for explaining